maestro-orchestra/src/test/resources/YamlCommandReaderTest/022_on_flow_start_complete.yaml

```diff
@@ -0,0 +1,7 @@
+appId: com.example.app
+onFlowStart:
+  - back
+onFlowComplete:
+  - scroll
```

Comment:
These are actually a set of Maestro commands only right?

Reply:
yes, correct, the subtype is `List<MaestroCommand>`